core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -2205,6 +2205,14 @@ private static RelDataType deriveTypeMapFromEntries(SqlOperatorBinding opBinding
           OperandTypes.NUMERIC_OPTIONAL_NUMERIC,
           SqlFunctionCategory.NUMERIC);
 
+  /** The "LOG(numeric1, numeric2)" function. Returns the base numeric1 logarithm of numeric. */
+  @LibraryOperator(libraries = {MYSQL, SPARK})
```

Comment:
the PR title just has MySQL.

Reply:
@tanclary How should I reasonably describe this PR title?
Add LOG function (enabled in MYSQL, Spark library) or Add LOG function (enabled in MYSQL* library).
Before I was Add LOG function (enabled in MYSQL, Spark library), I thought you asked me to change it to Add LOG function (enabled in MYSQL library), it seems I didn’t understand what you meant, sorry

Reply:
I use Mysql_Style to describe it. Do you think it is acceptable?

Reply:
No. Please look at other similar function implementations to see how they handle multiple libraries.

Reply:
Other functions do not specifically adapt their log functions to the mysql and spark dialects like the log function.

Reply:
I found similar cases Add SHA256, SHA512 functions (enabled in BigQuery and PostgreSQL libraries), so I should change back to Add LOG function (enabled in MYSQL, Spark library)

Reply:
"Style" doesn't add much. The issue is you can't name a function`LOG_MYSQL` if that function is also used for Spark. Does that make sense?

Reply:
LOG_MYSQL_SPARK?

Reply:
What I wrote at the beginning was LOG_MYSQL_SPARK. It seems that I have always misunderstood what you meant. Sorry.

Reply:
@tanclary PTAL, thank you